judge/admin/contest.py

```diff
@@ -148,7 +156,27 @@ def make_private(self, request, queryset):
 
     def get_urls(self):
         return [url(r'^rate/all/$', self.rate_all_view, name='judge_contest_rate_all'),
-                url(r'^(\d+)/rate/$', self.rate_view, name='judge_contest_rate')] + super(ContestAdmin, self).get_urls()
+                url(r'^(\d+)/rate/$', self.rate_view, name='judge_contest_rate'),
+                url(r'^(\d+)/judge/(\d+)/$', self.rejudge_view, name='judge_contest_rejudge'),
+                ] + super(ContestAdmin, self).get_urls()
```

Comment:
This hanging ] looks wonky.

Reply:
Agreed, please dedent the `]` to the same level as `return`.